core/sql/executor/ExHbaseAccess.cpp

```diff
@@ -2967,22 +2968,28 @@ void ExHbaseAccessTcb::handleException(NAHeap *heap,
                                     ComCondition *errorCond,
                                     ExpHbaseInterface * ehi,
                                     NABoolean & LoggingFileCreated,
-                                    char *loggingFileName)
+                                    char *loggingFileName,
+                                    ComDiagsArea **loggingErrorDiags)
 {
   Lng32 errorMsgLen = 0;
   charBuf *cBuf = NULL;
   char *errorMsg;
   Lng32 retcode;
 
+  if (*loggingErrorDiags != NULL)
```

Comment:
Basically, we want to skip the error rows in the source and continue to load. Load command has constructs to log the error row and the error information about it.  We just want to report the error while logging the error rows as warning at the end of the load command. This check is skipping the logging once an error is encountered  in the earlier attempt to log the error row.